src/netlog/netlog-network.c

```diff
@@ -139,13 +141,19 @@ int manager_push_to_network(Manager *m,
         IOVEC_SET_STRING(iov[n++], " ");
 
         /* Seventh: msgid */
-        IOVEC_SET_STRING(iov[n++], RFC_5424_NILVALUE);
+        if (msgid)
```

Comment:
Is there a need for making this feature an opt-in feature by providing a new configuration in /etc/systemd/syslogd-netlogd.conf that is off by default? Something like:
```
GetStructuredDataFromJournal=true
GetMessageIDFromJournal=true
```